src/common-model/functions.ts

```diff
@@ -172,23 +172,35 @@ export function getClinicalEntitySubmittedData(
   return clinicalRecords;
 }
 
+export const DonorCompletionFields: Array<keyof Donor | EntityAlias> = [
```

Comment:
`donorCompletionFields` it's just an array like `requiredFields` good to keep naming consistent